index.js

```diff
@@ -253,9 +265,7 @@ function getResponseStatusCode (res) {
  */
 
 function headersSent (res) {
-  return typeof res.headersSent !== 'boolean'
```

Comment:
Please revert it, let's try to make less unwanted changes so its easier for the folks to review